app/src/test/java/com/intive/selftraining/selftraining/data/MovieDetailsMapperTest.kt

```diff
@@ -0,0 +1,43 @@
+package com.intive.selftraining.selftraining.data
+
+import androidx.arch.core.executor.testing.InstantTaskExecutorRule
+import com.intive.selftraining.selftraining.movieDetails.model.MovieDetails
+import com.intive.selftraining.selftraining.network.models.listMovies.ImagesEntity
+import com.intive.selftraining.selftraining.network.models.movieDetails.MovieDetailsEntitiy
+import com.nhaarman.mockitokotlin2.mock
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.TestRule
+import org.mockito.Mockito.`when`
+
+class MovieDetailsMapperTest {
+
+    private val mapper = mock<MovieDetailsMapper>()
+    private val movieDetail: MovieDetails = getMovieDetails()
+    private val movieDetailsEntity = mock<MovieDetailsEntitiy>()
+    private val imagesEntity = mock<ImagesEntity>()
+
+    @Rule
+    @JvmField
+    val rule: TestRule = InstantTaskExecutorRule()
+
+    @Test
+    fun mapFromEntity() {
```

Comment:
maybe renaming the name of the method mapFromEntity() in this style: `map from entity`()

Reply:
Done!